### PR TITLE
[stable/nginx-ingress] Add helpful comment regarding affinity

### PR DESCRIPTION
**Pull request by naseemkullah**
I presume this is how people keep the pods when run as a deployment running on separate nodes? If so it's nice to have it in the values file to easily uncomment.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.11.1
+version: 1.11.2
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: This should be 1.11.3

**Review by ChiefAlexander — CHANGES_REQUESTED** [review]
Hey thanks for the PR! Just need to bump the version and I will be fine with this.

File: stable/nginx-ingress/Chart.yaml (hunk 1 above)
Comment (naseemkullah) [reply stable/nginx-ingress/Chart.yaml]: Oops, done! Thanks.

**Pull request merged**